qutebrowser/config/configdata.yml

```diff
@@ -493,17 +494,17 @@ content.javascript.prompt:
 content.local_content_can_access_remote_urls:
   default: false
   type: Bool
-  desc: Whether locally loaded documents are allowed to access remote urls.
+  desc: Allow locally loaded documents to access remote URLs.
```

Comment:
If we decide to enforce 'Enable' at the front of every bool, this should be changed. I like this more though, fwiw.

Reply:
I think we don't need 'Enable' for every bool as long as it's imperative mood.
@The-Compiler: Do you agree?

Reply:
Yeah, and for cases like that (or the JS permissions), I think "Allow ..." is better.